### PR TITLE
Don't know what to put here 

### PR DESCRIPTION
**Pull request by jamestang12**
Still working in progress, the main purpose of this branch is that everyone on the team would have a better picture of the current status of explore page since I am planning not to merge this as we don't have the fetching part done yet :)

### CODE REVIEW

**Comment by jamestang12** [comment]
Ignore the changes in ``App.tsx`` this is only for testing since we can't fetch image from db at the moment so this will make development easier

File: client/src/pages/Explore.tsx [hunk 1]
```diff
@@ -13,41 +13,162 @@ import {
...
+}
+
 export const Explore = () => {
     const store: UserStore = useStore();
     const user = store.user;
-    const images = user.images;
+    const images1 = user.images;
+    console.log(images1);
+
+    const dummyImage1 : Image = {
```
Comment (patel1643) [inline client/src/pages/Explore.tsx:53]: Make sure you clean this up once we fetch actual images lol

**Comment by jamestang12** [comment]
Also pls ignore ``const images1 = user.images;`` because I am testing it with dummy data and this is the only way I could visualize the data on the client

**Review by patel1643 — APPROVED** [review]
LGTM

File: client/src/pages/Explore.tsx (hunk 1 above)
Comment (jamestang12) [reply client/src/pages/Explore.tsx:53]: Yea I think I'll probably not gonna merge this for now, I'll wait for the fetching part to work so I can also test it with the back-end as well

Comment (patel1643) [reply client/src/pages/Explore.tsx:53]: yeah

Comment (jamestang12) [reply client/src/pages/Explore.tsx:53]: And by having this here so everyone on the team would have better picture of the currently process

File: client/src/pages/Explore.tsx [hunk 2]
```diff
@@ -48,100 +47,31 @@ export const Explore = () => {
...
+        } else {
+            tags.map(item => {
+                if(item.name.toLocaleLowerCase().includes(searchQuery.toLocaleLowerCase())) {
+                    newTagsList.push(item);
+                }
+            });
+        }
+        setDisplayTags(newTagsList);
+    };
+
```
Comment (patel1643) [inline client/src/pages/Explore.tsx:74]: More like copying Parth's code

**Review by patel1643 — APPROVED** [review]

**Pull request merged**